syntax = "proto3";

message Major_response {
    string jwt = 8 ;
}

